packages/devtools_app/lib/src/analytics/prompt.dart

```diff
@@ -0,0 +1,157 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+import '../theme.dart';
+import '../utils.dart';
+import 'provider.dart';
+
+/// Conditionally displays a prompt to request permission for collection of
+/// usage analytics.
+class AnalyticsPrompt extends StatefulWidget {
+  const AnalyticsPrompt({
+    @required this.provider,
+    @required this.child,
+  });
+
+  final Widget child;
+  final AnalyticsProvider provider;
+
+  @override
+  State<AnalyticsPrompt> createState() =>
+      _AnalyticsPromptState(provider, child);
+}
+
+class _AnalyticsPromptState extends State<AnalyticsPrompt> {
+  _AnalyticsPromptState(this._provider, this._child);
+
+  final Widget _child;
+  final AnalyticsProvider _provider;
+
+  bool _isVisible = false;
+
+  @override
+  void initState() {
+    super.initState();
+    () async {
+      try {
+        await _provider.initialize();
+
+        if (_provider.isGtagsEnabled) {
+          if (await _provider.isFirstRun) {
+            setState(() {
+              _isVisible = true;
+            });
+          } else if (await _provider.isEnabled) {
+            _provider.setUpAnalytics();
+          }
+        }
+      } catch (e) {
+        print('Error collecting analytics:\n$e');
+      }
+    }();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Expanded(
```

Comment:
can these two widgets be removed? I don't think we need the extra column because it just has a single child, (Expanded) which also has a single child